### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by boomanaiden154**
I made some additions to the documentation pages based on some of the issues that I have come across. The additions I have made are as follows:
- Added documentation for multipath configuration on Ubuntu 20.04
- Added line in the `storage.md` file in the LVM pool configuration section to indicate that volume snapshots on volumes provisioned through an LVM pool are not supported.

### CODE REVIEW

File: doc/host-setup.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ following options to load DRBD:
...
+
+In Ubuntu 20.04, the multipath daemon opens up DRBD devices if it is not configured which prevents Piraeus from working properly.
+To configure the multipath daemon, put the following into the file `/etc/multipath/conf.d/drbd.conf`. You will need to create the
+directory `/etc/multipath/conf.d` on new installations of Ubuntu 20.04.
+
+```
+blacklist {
+        devnode "^drbd[0-9]+"
+}
+```
```
Comment (WanzenBug) [inline doc/host-setup.md:23]: I'd rephrase this section a bit: the problem with `multipathd` doesn't only exist on ubuntu 20.04, but basically in any default multipath installation. The difference is that Ubuntu 20.04 has `multipathd` installed and enabled by default.

So in essence I'd change this section to say "If you are using `multipathd`..." and add that Ubuntu 20.04 enables that by default as a note.

**Review by WanzenBug — CHANGES_REQUESTED** [review]
Thank you for improving our docs. I'd suggest to change the section on multipathd a bit.

File: doc/host-setup.md (hunk 1 above)
Comment (boomanaiden154) [reply doc/host-setup.md:24]: I just adjusted the section to be labelled "Software compatibility" instead of making it distro specific and I updated the multipath section to account for the changes that you requested. It should be more general now for all multipath installations.

**Review by WanzenBug — APPROVED** [review]
Thanks again, changes look good.

Please squash the changes into one commit and use `--signoff` to indicate that the contribution is your own work.

**Review by WanzenBug — APPROVED** [review]
